### PR TITLE
Fix misc typos and grammar

### PR DESCRIPTION
**Pull request by thejchap**
1st of a couple :+1:

### CODE REVIEW

File: lib/jscs-rules/disallow-multiple-var-decl-with-assignment.js [hunk 1]
```diff
@@ -20,8 +20,8 @@ module.exports.prototype = {
 
   check: function(file, errors) {
     file.iterateNodesByType('VariableDeclaration', function(node) {
-      // allow multiple var declarations in for statement
-      // for (var i = 0, j = myArray.length; i < j; i++) {}
+      // Allow multiple var declarations in for statement
+      // for (var i = 0, j = myArray.length; i < j; i++) {}.
```
Comment (rwjblue) [inline lib/jscs-rules/disallow-multiple-var-decl-with-assignment.js]: this is not a sentence, I wouldn't think we would put a period here

File: packages/ember-application/tests/system/dependency_injection/default_resolver_test.js [hunk 2]
```diff
@@ -126,7 +126,7 @@ QUnit.test('the default resolver resolves helpers on the namespace', function()
 
   expectDeprecation(function() {
     LegacyHTMLBarsBoundHelper = makeHTMLBarsBoundHelper(function() {});
-  }, 'Using `Ember.HTMLBars.makeBoundHelper` is deprecated. Please refactor to using `Ember.Helper` or `Ember.Helper.helper`.');
```
Comment (rwjblue) [inline packages/ember-application/tests/system/dependency_injection/default_resolver_test.js]: This is testing that a specific deprecation message is issued, if you change the expected message you also need to change where it is triggered.

**Comment by rwjblue** [comment]
Looks good. I think the failing tests are related to the deprecation message change I pointed out inline.

**Comment by thejchap** [comment]
@rwjblue thanks...made the changes, although `TEST_SUITE=each-package-tests` is still failing and I can't really decipher why...any thoughts? https://travis-ci.org/emberjs/ember.js/jobs/105594858

**Comment by rwjblue** [comment]
Phantom crashed on the VM. I restarted it.

**Comment by thejchap** [comment]
@rwjblue strange...same thing again

**Comment by thejchap** [comment]
@rwjblue not sure what's going on here, is this just Phantom crashing again?

**Comment by rwjblue** [comment]
Yeah, likely just phantom crashing. @chancancode just did a bit of work which seems to be helping with the phantom crashes. If you rebase, you can grab those updates.

**Comment by thejchap** [comment]
@rwjblue :+1:

**Comment by thejchap** [comment]
@rwjblue anything else needed here?

**Comment by thejchap** [comment]
@rwjblue ping :]

**Comment by stefanpenner** [comment]
thank you for the contributions :)

**Pull request merged**